Gemfile

```diff
@@ -1,4 +1,3 @@
 source 'https://rubygems.org'
-gem 'fakes3', :path => '.' # for dev and test, use local fakes3
```

Comment:
To fix:
```
Your Gemfile lists the gem fakes3 (>= 0) more than once.
You should probably keep only one of them.
While it's not a problem now, it could cause errors if you change the version of one of them later.

[!] There was an error parsing `Gemfile`: You cannot specify the same gem twice coming from different sources.
You specified that fakes3 (>= 0) should come from source at `.` and source at `.`
. Bundler cannot continue.

 #  from /app/Gemfile:4
 #  -------------------------------------------
 #  # Specify your gem's dependencies in fakes3.gemspec
 >  gemspec
 #  -------------------------------------------
The command '/bin/sh -c bundle install' returned a non-zero code: 4
```